src/server.js

```diff
@@ -1,168 +1,161 @@
 #!/usr/bin/env node
 
 const fs = require('fs');
-const koa = require('koa');
+const Koa = require('koa');
+const puppeteer = require('puppeteer');
 const tmp = require('tmp');
 const winston = require('winston');
-const createPhantomPool = require('phantom-pool').default;
-
-const logger = new (winston.Logger)({
-    level: process.env.SCREENIE_LOG_LEVEL || 'info',
-    transports: [
-        new (winston.transports.Console)({
-            timestamp: () => new Date().toISOString(),
-        }),
-    ],
-});
 
-const app = koa();
-const pool = createPhantomPool({
-    min: process.env.SCREENIE_POOL_MIN || 2,
-    max: process.env.SCREENIE_POOL_MAX || 10,
+const logger = new winston.Logger({
+  level: process.env.SCREENIE_LOG_LEVEL || 'info',
+  transports: [
+    new winston.transports.Console({
+      timestamp: () => new Date().toISOString()
+    })
+  ]
 });
+
+const app = new Koa();
 const serverPort = process.env.SCREENIE_PORT || 3000;
 
 const imageSize = {
-    width: process.env.SCREENIE_WIDTH || 1024,
-    height: process.env.SCREENIE_HEIGHT || 768,
+  width: process.env.SCREENIE_WIDTH || 1024,
+  height: process.env.SCREENIE_HEIGHT || 768
```

Comment:
Same here with ending comma. So go through the whole code and update with that. Maybe it's easy to just run prettier on it?

Reply:
Or maybe you have run prettier already, but just not with the `"trailingComma": "es5",` option? Can add a .prettierrc file to the repo with it.